src/Tizen.WebView/Tizen.WebView/PolicyDecisionEventArgs.cs

```diff
@@ -0,0 +1,250 @@
+    /// <since_tizen> 6 </since_tizen>
+    public abstract class PolicyDecisionEventArgs : EventArgs
+    {
+        private IntPtr _handle;
+
+        public PolicyDecisionEventArgs(IntPtr handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Gets the Url.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Url
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_url_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets the scheme.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Scheme
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_scheme_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Ignores the action which triggers this decision.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Ignore()
+        {
+            Interop.ChromiumEwk.ewk_policy_decision_ignore(_handle);
+        }
+
+        /// <summary>
+        /// Accepts the action which triggers this decision.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Use()
+        {
+            Interop.ChromiumEwk.ewk_policy_decision_use(_handle);
+        }
+
+    }
+
+    public class NewWindowPolicyEventArgs : PolicyDecisionEventArgs 
+    {
+        private IntPtr _handle;
+        public NewWindowPolicyEventArgs(IntPtr handle) : base(handle)
+        {
+            _handle = handle;
+        }
+        
+        internal static NewWindowPolicyEventArgs CreateFromSmartEvent(IntPtr data, IntPtr obj, IntPtr info)
+        {
+            return new NewWindowPolicyEventArgs(info);
+        }
+    }
+
+
+    public class NavigationPolicyEventArgs : PolicyDecisionEventArgs
+    {
+        private IntPtr _handle;
+        public NavigationPolicyEventArgs(IntPtr handle) : base(handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Gets the Navigation Type.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public NavigationType NavigationType
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_navigation_type_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets the cookie.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Cookie
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_cookie_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Checks if frame requested in policy decision is main frame.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsMainFrame()
```

Comment:
Plz move this method to `PolicyDecisionEventArgs`, it should be provided for all of policy event.

Reply:
Done